uber/model_checks.py

```diff
@@ -197,14 +249,17 @@ def job_conflicts(job):
                 return 'You cannot change this job to this time, because {} is already working a shift then'.format(shift.attendee.full_name)
 
 
-cashformpoints_amount = money_amount
-
+@validation.OldMPointExchange
 def oldmpointexchange_numbers(mpe):
     if not str(mpe.amount).isdigit():
         return 'MPoints must be a positive integer'
 
-sale_required = [('what',"What's being sold")]
-def sale_amounts(sale):
+
+Sale.required = [('what', "What's being sold")]
+
+
+@validation.Sale
+def cash_and_mpoints(sale):
     if not str(sale.cash).isdigit() or int(sale.cash) < 0:
         return 'Cash must be a positive integer'
     if not str(sale.mpoints).isdigit() or int(sale.mpoints) < 0:
```

Comment:
From Github's documentation:

> If your diffs exceed the limits and can't be viewed online, you can always pull the changes locally and view the diff there. It may not be as convenient for you, but it'll get the job done.

Apparently ``models.py`` has too many changes to show!

It's... mostly the same, honestly.  I mean, there were a bunch of refactors, to support things like ``@cost_property`` and ``@presave_adjustment`` and some of the new overridable properties like ``addons``.  Plus we changed the directionality of our ``relationship`` declarations.  Plus we touched a bunch of stuff with the ``c`` refactor.  Plus we added some ``Column`` parameters for declaring which fields are ``admin_only``.  Plus a few other things, which I believe are all/mostly covered in the MAGStock review: https://github.com/magfest/magstock/pull/3/files

So anyway, you won't be able to look at the diff here, but you can always look at the file and ask questions as necessary.  Or pull down the diff and look at it locally on your machine, either as plaintext or some some sort of diff compare tool.